### PR TITLE
Feature of static participant list

### PR DESCRIPTION
**Pull request by Gemakk**
## Purpose

* Enable viewing a static Participant List

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## Test the code
* Ensure participant list correctly represents the current list of participants in the call
* Ensure a full list of participants only takes up half the screen and is scrollable

### CODE REVIEW

**Comment by lpx1989** [comment]
Tested the basic scenarios. While I noticed there is a little difference with the Figma design for the position of the accessoryImage. In the Figma design the image is not outside of end of separator line. Should we reduce the marginEnd of separator line to align it with accessory Image or increase the marginEnd for the accessory Image?
Screenshot_20210519-114707

**Comment by Gemakk** [comment]
> Tested the basic scenarios. While I noticed there is a little difference with the Figma design for the position of the accessoryImage. In the Figma design the image is not outside of end of separator line. Should we reduce the marginEnd of separator line to align it with accessory Image or increase the marginEnd for the accessory Image?

Thanks for the catch!  I've removed the margin

File: AzureCalling/app/src/main/java/com/azure/samples/communication/calling/helpers/ParticipantInfo.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.samples.communication.calling.helpers;
+
+public class ParticipantInfo {
+    private String displayName;
+    private Boolean ismuted;
+
+    public ParticipantInfo(final String displayName, final Boolean ismuted) {
```
Comment (iaulakh) [inline AzureCalling/app/src/main/java/com/azure/samples/communication/calling/helpers/ParticipantInfo.java:10]: isMuted

**Comment by iaulakh** [comment]
when new participant joins and participant list is already open, then newly added participant not displayed. is it expected?
Screen Shot 2021-05-31 at 8 18 27 AM

File: AzureCalling/app/src/main/java/com/azure/samples/communication/calling/helpers/ParticipantListAdapter.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import android.content.Context;
+
+import androidx.core.content.ContextCompat;
+
+import com.azure.samples.communication.calling.R;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ParticipantListAdapter extends BottomCellAdapter {
```
Comment (iaulakh) [inline AzureCalling/app/src/main/java/com/azure/samples/communication/calling/helpers/ParticipantListAdapter.java:15]: are we going to have only one BottomCellAdapter?  If we need multiple bottom cell in app, probably we can add the indicator to name `ParticipantListBottomCellAdapter`

**Comment by Gemakk** [comment]
> when new participant joins and participant list is already open, then newly added participant not displayed. is it expected?

That's correct.  The list will be updated once the dynamic list story is complete.

**Review by iaulakh — DISMISSED** [review]

File: AzureCalling/app/src/main/java/com/azure/samples/communication/calling/helpers/ParticipantListAdapter.java (hunk 2 above)
Comment (Gemakk) [reply AzureCalling/app/src/main/java/com/azure/samples/communication/calling/helpers/ParticipantListAdapter.java:15]: We're going to have at least three: one for Audio Selection, Participant List, and one for the additional options menu.  Here you can see the current extensions:
image

**Review by pavelprystinka — DISMISSED** [review]

**Review by pavelprystinka — APPROVED** [review]

**Review by iaulakh — APPROVED** [review]

**Pull request merged**